core/includes/ajax.inc

```diff
@@ -590,7 +590,7 @@ function ajax_prepare_response($page_callback_result) {
         break;
 
       case MENU_SITE_OFFLINE:
-        $commands[] = ajax_command_alert(filter_xss_admin(t(config_get('system.core', 'maintenance_mode_message'), array('@site' => config_get('system.core', 'site_name')))));
+        $commands[] = ajax_command_alert(filter_xss_admin(token_replace(config_get('system.core', 'maintenance_mode_message'))));
```

Comment:
Not sure about the removal of t() here. I think it should be left in place inside the token_replace. `token_replace(t($message))`.